javalib/src/main/scala/java/nio/file/Path.scala

```diff
@@ -35,3 +35,20 @@ trait Path extends Comparable[Path] with Iterable[Path] with Watchable {
   def toString(): String
   def toUri(): URI
 }
+
+// https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/nio/file/Paths.html
```

Comment:
1) Wonderful to see a test. It increases confidence.

2) I suggest dropping the entire block comment and
    adding a single line stating that "Path.of()" is introduced
    in Java 11.

    To my thinking, which is not the only approach. The test
    file should be concerned if its area of interest, Path, works     
    and not introduce possible confusion or interaction    
    (unless the latter is tested). Then again, I switch between
    so many topics I get confused easily.

    The effective point of intervention (education early in the  
     chain) is in "Paths".